src/styles/main.scss

```diff
@@ -0,0 +1,129 @@
+$bodyfont: Roboto, sans-serif;
+
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
These styles should be moved to the `reset or normalize` SCSS file because they are global styles and it's a good practice to keep them separate from component styles.